### PR TITLE
doc: fix 404 link when link to install-dependencies

### PR DESCRIPTION
**Pull request by lilien1010**
### Summary

doc: fix 404 link when link to install-dependencies

### CODE REVIEW

**Comment by membphis** [comment]
Thank you for your contribution, we will merge this PR later.

**Comment by moonming** [comment]
fixed https://github.com/iresty/apisix/issues/705

File: lua/apisix/schema_def.lua [hunk 1]
```diff
@@ -236,7 +236,7 @@ local upstream_schema = {
         key = {
             description = "the key of chash for dynamic load balancing",
             type = "string",
-            enum = {"remote_addr"},
+            pattern = [[^((uri|server_name|server_addr|request_uri|remote_port|remote_addr|query_string|host|hostname)|arg_[0-9a-zA-z_-]+)$]]
```
Comment (moonming) [inline lua/apisix/schema_def.lua]: please add another new PR for this commit. We should only fix one thing per PR.

**Comment by juzhiyuan** [comment]
any update?

**Comment by membphis** [comment]
will merge this PR later

**Pull request closed without merging**